<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ss_PartFrame</class>
 <widget class="QFrame" name="ss_PartFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QToolButton" name="btn_selectNon">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_selectRev">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>68</y>
     <width>321</width>
     <height>781</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::South</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>零件</string>
    </attribute>
    <widget class="QListWidget" name="partList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>761</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>板材</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>271</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#00ffff;</string>
   </property>
   <property name="text">
    <string>排样:零件</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_selectAll">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>40</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
